srcs/request_handling/FixedResponses.cpp

```
@@ -1,124 +1,99 @@
 	auto response_packet = std::make_shared<ResponsePacket>();
-	response_packet->set_status_code(500);
-	response_packet->set_status_message("Internal Server Error");
+	response_packet->set_status_code(status_code);
+	response_packet->set_status_message(status_message);
 	response_packet->set_final_response();
 	response_packet->setResponseReady(true);
 	return response_packet;
 }
 
-std::shared_ptr<ResponsePacket> bad_request()
+std::shared_ptr<ResponsePacket> dummy_response()
```

cppcheck

[style] The function 'dummy_response' is never used.

(unusedFunction)